ios/RNTPTDocumentView.m

```diff
@@ -2248,6 +2248,16 @@ - (void)setBackgroundColor:(NSDictionary *)backgroundColor
     }
 }
 
+- (void)showViewModeDialog
+{
```

Comment:
What happens if you call this method when it's already being presented? I would make this method a no-op if the dialog is already being presented, and document this fact.

```suggestion
{

    if( [self.currentDocumentViewController.settingsViewController isBeingPresented] )
    {
        return;
    }
```